demos/middleware/next.config.js

```diff
@@ -6,6 +6,7 @@ const nextConfig = {
     // your project has ESLint errors.
     ignoreDuringBuilds: true,
   },
+  generateBuildId: () => 'build-id',
```

Comment:
[dust] Consider leaving. a comment linking to https://nextjs.org/docs/api-reference/next.config.js/configuring-the-build-id